website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,16 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+
+        // Adapted from http://www.quirksmode.org/js/cookies.html
+        var cookies = document.cookie.split(';');
+        for (var i = 0; i < cookies.length; i++) {
+          var cookieName = cookies[i].match(/([^=]+)(?=\=)/);
+          if(cookieName != null){
+            document.cookie = cookieName[0] + '= ; expires=Thu Jan 1 00:00:00 1970 GMT; path=/';
+          }
+        }
+
         $window.location.href = '/logout';
       };
 
```

Comment:
Please also edit the texts that are on the clear-browser-data page. 

Change this: "... click the button below to clear local storage for this website ..." 
to this: "... click the button below to clear local storage and most cookies for this website ..." 

Change the button to say "Clear Data" and its on-hover text to say "This button will clear local storage and most cookies, and log you out." (note the new full stop at the end).


The pop-up message that appears after clicking the button should be changed to:

"Clearing Data"

"Habitica's stored data is being cleared from your browser. You will be logged out and redirected to the home page. Please wait."


Show us screenshots of what the page and the pop-up message look like after that, and then it should be good to merge!